scripts/updateapilist.py

```diff
@@ -159,13 +201,29 @@ def all_clients() -> List[CloudClient]:
         repositories = response.json().get("items", [])
         if len(repositories) == 0:
             break
-        clients.extend(get_clients_batch_from_response_json(repositories))
+
+        split_repo_extractor = Extractor(path_format=SPLIT_REPO_PATH_FORMAT, response_key=REPO_RESPONSE_KEY)
+        return split_repo_extractor.get_clients_from_batch_response(repositories)
+
+
+def get_token():
+    if 'GITHUB_TOKEN' not in os.environ:
+        raise MissingGithubToken("Please include a GITHUB_TOKEN env var.")
+    
+    token = os.environ['GITHUB_TOKEN']
+    return token
```

Comment:
We could potentially remove the requirement to query the Github API as the list of split repositories will remain static for the most part (we won't create new GitHub repositories for clients moving forward since new clients are generated in `google-cloud-python`).  We could make this change in a follow up PR

Reply:
@parthea Yes this will be addressed in a follow-up PR.